src/commands/test.rs

```diff
@@ -208,6 +230,43 @@ fn run_pipeline(
     )
 }
 
+async fn update_score_badge_in_file(
+    path: PathBuf,
+    competition: Option<String>,
+    score: &Py<PyAny>,
+    url: Url,
+) -> Result<()> {
+    let badge_url = build_shield_score_badge(
+        score,
+        url.join(&competition.map_or_else(
+            || "competitons".to_string(),
+            |name| format!("competitions/{}", name),
+        ))?,
+    )?;
+    with_locked_file(
+        |file| {
+            async move {
+                let mut contents = String::new();
+                file.read_to_string(&mut contents).await?;
+                let updated_content = regex_replace_all!(
+                    r"(?:<!--\s*aqora:score:start\s*-->.*?<!--\s*aqora:score:end\s*-->)|(!\[Aqora Score Badge\]\([^\)]+\))",
```

Comment:
Would this replace `<!-- aqora:score:start -->![Aqora Score Badge](...)<!-- aqora:score:end -->` with `![Aqora Score Badge](...)` or am I reading this wrong? Shouldn't it wrap the the replaced section in comments as well?

Reply:
Nope! This is going to replace `<!-- aqora:score:start --><!-- aqora:score:end -->` or `![Aqora Score Badge](...)`with the generated badge. The thing is, we have to either comment out or replace an existing badge that we don’t want to keep!